chutney/action-impl/src/test/java/com/chutneytesting/action/kafka/KafkaBasicConsumeActionIntegrationTest.java

```diff
@@ -68,92 +82,204 @@ public void before() {
     }
 
     @Test
-    public void should_consume_message_from_broker_without_truststore() {
-
-        // given
+    void consume_message_from_broker() {
+        // Given
         producer.send(new ProducerRecord<>(uniqueTopic, 123, "my-test-value"));
 
         Map<String, String> props = new HashMap<>();
         props.put(ConsumerConfig.GROUP_ID_CONFIG, GROUP);
         props.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, OffsetResetStrategy.EARLIEST.name().toLowerCase());
 
+        // When
         Action consumeAction = getKafkaBasicConsumeAction(targetBuilder.build(), props, false);
-
-
-        // when
         ActionExecutionResult actionExecutionResult = consumeAction.execute();
 
-        // then
+        // Then
         assertThat(actionExecutionResult.status).isEqualTo(Success);
         List<Map<String, Object>> body = assertActionOutputsSize(actionExecutionResult, 1);
         assertThat(body.get(0).get("payload")).isEqualTo("my-test-value");
     }
 
-    @Test
-    public void consumer_from_target_with_truststore_should_reject_ssl_connection_with_broker_without_truststore_configured() throws URISyntaxException {
-        // given
-        String keystore_jks = Paths.get(requireNonNull(HttpsServerStartActionTest.class.getResource("/security/server.jks")).toURI()).toAbsolutePath().toString();
-        Target target = targetBuilder.withProperty("trustStore", keystore_jks)
-            .withProperty("trustStorePassword", "server")
-            .withProperty("security.protocol", "SSL")
-            .build();
+    @ParameterizedTest
+    @CsvSource(textBlock = """
+        /security/truststore.jks,truststore
+        /security/truststore_empty_pass.jks,''
+        /security/truststore_empty_pass.jks,
+        """)
+    void consumer_from_target_with_truststore_should_reject_ssl_connection_with_broker_without_ssl_configured(String truststorePath, String truststorePass) throws URISyntaxException {
```

Comment:
why 3 case for this failing test case. I prefer to have this on nominal test case

Reply:
The embedded broker does not allow to do SSL.